components/org.wso2.carbon.identity.application.authenticator.push.device.handler/pom.xml

```diff
@@ -243,36 +241,34 @@
-                           org.wso2.carbon.user.core.service;version="[4.6,5)",
-                           org.wso2.carbon.utils;version="[4.6,5)"
-                       </Import-Package>
-                        <!--<DynamicImport-Package>*</DynamicImport-Package>-->
+                        <Import-Package>
+                            com.fasterxml.jackson.core;version="[2.9,3)",
+                            com.fasterxml.jackson.databind;version="[2.9,3)",
+                            com.yubico.internal.util,org.apache.commons.lang;version="[2.6,3)",
+                            org.apache.commons.logging;version="[ 1.2,2)",
+                            org.osgi.framework;version="[1.8,2)",
+                            org.wso2.carbon.context;version="[4.6,5)",
+                            org.wso2.carbon.identity.application.authentication.framework.store;version="[5.12,6)",
+                            org.wso2.carbon.identity.application.authentication.framework.exception;version="[5.12,6)",
+                            org.wso2.carbon.identity.application.common.cache;version="[5.12,6)",
+                            org.wso2.carbon.identity.application.common.model;version="[5.12,6)",
+                            org.wso2.carbon.identity.base;version="[5.12,6)",
+                            org.wso2.carbon.identity.core.util;version="[5.12,6)",
+                            org.wso2.carbon.identity.user.store.configuration.listener;version="[5.12,6)",
+                            org.wso2.carbon.user.api;version="[1.0,2)",
+                            org.wso2.carbon.user.core.common;version="[4.6,5)",
+                            org.wso2.carbon.user.core.service;version="[4.6,5)",
+                            org.wso2.carbon.utils;version="[4.6,5)"
+                        </Import-Package>
                         <Export-Package>
-                            !org.wso2.carbon.identity.application.authenticator.biometric.device.handler.internal,
-                            org.wso2.carbon.identity.application.authenticator.biometric.device.handler;
-                            org.wso2.carbon.identity.application.authenticator.biometric.device.handler.exception;
-                            org.wso2.carbon.identity.application.authenticator.biometric.device.handler.dao;
-                            org.wso2.carbon.identity.application.authenticator.biometric.device.handler.model;
-                            org.wso2.carbon.identity.application.authenticator.biometric.device.handler.util;
-                            org.wso2.carbon.identity.application.authenticator.biometric.device.handler.impl;
-                            org.wso2.carbon.identity.application.authenticator.biometric.device.handler.cache
+                            !org.wso2.carbon.identity.application.authenticator.push.device.handler.internal,
+                            org.wso2.carbon.identity.application.authenticator.push.device.handler;
```

Comment:
@pulasthi7 is this exporting the whole package?

Reply:
only the package classes, not subpackages (should use .* for that)